gamestonk_terminal/settings_controller.py

```diff
@@ -28,6 +28,7 @@ class SettingsController(BaseController):
     """Settings Controller class"""
 
     CHOICES_COMMANDS: List[str] = [
+        "logcollection",
```

Comment:
Missing `ONLY_SEND_FILE_SIZE` feature flag.

Tested and it worked for LOG_COLLECTION!

Reply:
`ONLY_SEND_FILE_SIZE`  flag will probably be removed in the future.

For now I just put what we will keep.